delivery_schenker/models/delivery_carrier.py

```diff
@@ -358,12 +358,27 @@ def _schenker_shipping_information_package_volume(self, picking, package):
             ]
         )
 
+    def _schenker_float_round(self, value, precision_digits):
+        """
+        The schenker api requires 2 decimal points.
+        Round up to avoid 0 if <0.005.
+        round -> repr -> float to avoid extra decimals that error with the Schema.
+        float_round(2.086, precision_digits=2) = 2.0100000000000002
```

Comment:
Your example doesn't look right
```suggestion
        float_round(2.086, precision_digits=2) = 2.0899999999999999
```

Reply:
Seems like I wrote the comment wrong, in the PR description the issue was 2.0086, not 2.086
![image](https://github.com/OCA/delivery-carrier/assets/47854752/2d8fc81f-c70e-41fe-9068-2827907b7b5b)
I will change it